docs/en/ingest-management/elastic-agent/install-fleet-managed-elastic-agent.asciidoc

```diff
@@ -30,24 +30,16 @@ quick start guides:
 Integrations assets are shared across spaces, users need the {kib} privileges in
 all spaces.
 
-* A {fleet-server} must be running in a location accessible to the {agent}. See
-<<fleet-server>>.
+* <<fleet-server,{fleet-server}>> must be running in a location accessible to
+the {agent}, and the {agent} must have a direct network connection to
+{fleet-server} and {es}. If you're using our hosted {ess} on {ecloud},
```

Comment:
Should "the Elastic Agent" here be either "Elastic Agent" or "the agent" (since `{agent}` resolves to `Elastic Agent`)? From what I've seen, in most places we don't use an article with "Elastic Agent", is that right?

Reply:
I think we had some places where we replaced "the agent" with "the Elastic Agent" but should have removed the article when we did that.

I think since we capitalize both words in Elastic Agent, we should not use "the" before it.